packages/strapi-plugin-content-manager/admin/src/components/MediaPreviewList/index.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isArray, includes, isEmpty } from 'lodash';
+
+import DefaultIcon from '../../assets/images/media/na.svg';
+
+import {
+  StyledMediaPreviewList,
+  MediaPreviewFile,
+  MediaPreviewImage,
+  MediaPreviewItem,
+  MediaPreviewText,
+} from './StyledMediaPreviewList';
+
+function MediaPreviewList({ hoverable, files }) {
+  const baseUrl = strapi.backendURL;
```

Comment:
I think that if the user is using an upload provider we don't have to include the backendURL

```
const getSrc = fileURL => fileURL.startsWith('/') ? `${strapi.backendURL}${fileURL}` : fileURL;
```